account/views.py

```diff
@@ -102,4 +102,29 @@ def post(self, request):
         new_access_token = str(RefreshToken(refresh_token).access_token)
         response = Response({"detail": "token refreshed"}, status=status.HTTP_200_OK)
         response.set_cookie("access_token", value=str(new_access_token), httponly=True)
-        return response
\ No newline at end of file
+        return response
+    
+class LogoutView(APIView):
+    @swagger_auto_schema(
+        operation_id="로그아웃",
+        operation_description="refresh token을 삭제합니다.",
+        request_body=LogoutRequestSerializer,
+        responses={204: "No Content", 400: "Bad Request", 401: "Unauthorized"},
+    )
+    def post(self, request):
+        refresh_token = request.data.get("refresh")
+
```

Comment:
여기서도 사용자의 로그인 여부를 먼저 확인해 주면 좋을 것 같아요 ~
```suggestion
        if not request.user.is_authenticated:
            return Response(
                {"detail": "please signin"}, status=status.HTTP_401_UNAUTHORIZED
            )
```

Reply:
그렇네요, 감사합니다!